lib/dropdown_search.dart

```diff
@@ -647,7 +658,9 @@ class DropdownSearchState<T> extends State<DropdownSearch<T>> {
         isScrollControlled: true,
         shape: widget.popupShape,
         context: context,
+        clipBehavior: widget.clipBehavior,
         builder: (ctx) {
+          if (widget.onOpenBottomSheet != null) widget.onOpenBottomSheet!(ctx);
```

Comment:
Could you explain the reasons behind this callback?
Why it exists only for bottom sheet but not for  the other types of modals?

Reply:
I'm sorry, I just want to pull first commit, the other commits are still unfinished-product.

Reply:
I need this callback to get the context so that I can update view on bottom sheet by this context.
Is there any good way to update view on bottom sheet? Such as, I want to update a [suffixIcon] on TexiField in the  bottom sheet in diffrent situation( like TexiField.text is empty or not ).
thank you for comment.